scalding-core/src/main/scala/com/twitter/scalding/ArgHelp.scala

```diff
@@ -0,0 +1,106 @@
+package com.twitter.scalding
+
+sealed abstract class DescribedArg {
```

Comment:
Why abstract class than abstract trait? seems an unusual choice. No real problem with it...just an unusual pattern